sql/wsrep_sst.cc

```diff
@@ -89,18 +89,21 @@ static Regex sst_allowed_methods_regex;
    characters and the ways they could be used to inject the command to
    the OS. However this approach seems to be too error prone.
    Instead of this we will just allow alpha-num + a few special characters
-   (colon, slash, dot, underscore, square brackets, hyphen).
+   (colon, slash, dot, underscore, square brackets, hyphen, '@' symbol).
 
-   It is the same as ^[\w:/.[\]-]+$    */
+   Note: '@' symbol is possible with encrypted rsync SST, where data can be
```

Comment:
Could you extend galera_sst_allowed_methods.test, please?

Reply:
Sure. I'll update it.

Reply:
Done